lib/project.ts

```diff
@@ -19,6 +19,7 @@ export class Project implements Project.IProject {
 		"kendoui.blank": "KendoUI.Empty",
 		"javascript.blank": "Blank"
 	};
+	private static WRONG_IOS_STATUS_BAR_DESCRIPTION = "Hidden";
```

Comment:
The name of this field seems off - there's nothing wrong or descriptive about the word "Hidden" 😺